### PR TITLE
Update docs | Security, Contributing, Getting Started (closes #3343)

### LINKED ISSUE #3343: docs: Link official Katacoda tutorial in quick-start
https://katacoda.com/bwplotka/courses/thanos/1-globalview

I just went through the tutorial and it is broken at the very beginning.  The .yml files are not files, but instead directories.  Once I deleted the directories and touched new files, I was able to proceed through the tutorial without issue.

### PR DESCRIPTION
**Pull request by conorevans**
* [ ] I added CHANGELOG entry for this change.
* [x] Change is not relevant to the end user.

## Changes

- Mostly grammar changes

- Updated link to Katacoda course, closing #3343

## Verification

N/A

Note: I hope this is not too trivial. I was mostly reading through the docs and hoped these changes would improve the overall readability and comprehension of Thanos.

### CODE REVIEW

**Comment by conorevans** [comment]
The errors:

lint-check

```
FILE: ./tutorials/katacoda/thanos/2-lts/step4.md
[✖] https://[[HOST_SUBDOMAIN]]-19095-[[KATACODA_HOST]].environments.katacoda.com/new/loaded
[✖] https://[[HOST_SUBDOMAIN]]-9091-[[KATACODA_HOST]].environments.katacoda.com/new/graph?g0.expr=&g0.tab=0&g0.stacked=0&g0.range_input=1h&g0.max_source_resolution=0s&g0.deduplicate=1&g0.partial_response=0&g0.store_matches=[]

3 links checked.

ERROR: 2 dead links found!
[✖] https://[[HOST_SUBDOMAIN]]-19095-[[KATACODA_HOST]].environments.katacoda.com/new/loaded → Status: 0
[✖] https://[[HOST_SUBDOMAIN]]-9091-[[KATACODA_HOST]].environments.katacoda.com/new/graph?g0.expr=&g0.tab=0&g0.stacked=0&g0.range_input=1h&g0.max_source_resolution=0s&g0.deduplicate=1&g0.partial_response=0&g0.store_matches=[] → Status: 0
```

Looks like that's unrelated.


```
Error: Makefile:234: recipe for target 'test-e2e' failed
Error: Process completed with exit code 2.
```

I'm not sure how this doc MR could possibly impact e2e tests. I've checked the folder and that uncertainty was reconfirmed, so if I've missed something please enlighten me - I'm happy to learn. Thanks

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,37 +2,37 @@
 
 This document explain the process of contributing to the Thanos project.
 
-First of all please follow the [CODE_OF_CONDUCT](/CODE_OF_CONDUCT.md) in all your interactions with the project.
+First of all please follow the [CODE_OF_CONDUCT](/CODE_OF_CONDUCT.md) in all your interactions within the project.
 
 ## Thanos Philosophy
 
-The philosophy of Thanos and our community is borrowing much from UNIX philosophy and the golang programming language.
+The philosophy of Thanos and our community borrows heavily from UNIX philosophy and the Golang programming language.
```
Comment (bwplotka) [inline CONTRIBUTING.md:9]:
```suggestion
The philosophy of Thanos and our community borrows heavily from UNIX philosophy and the Go programming language.
```

:hugs:  =D to be strict.

**Review by bwplotka — APPROVED** [review]
:muscle: 

Thank you! Definitely not too trivial, why it would be trivial? :hugs:

File: CONTRIBUTING.md (hunk 1 above)
Comment (bwplotka) [reply CONTRIBUTING.md:9]: Ups, merged already. (: We can fix later.

**Comment by bwplotka** [comment]
> I'm not sure how this doc MR could possibly impact e2e tests. I've checked the folder and that uncertainty was reconfirmed, so if I've missed something please enlighten me - I'm happy to learn. Thanks

Yup, flaky ):

**Pull request merged**